src/com/facebook/buck/ide/intellij/model/folders/AbstractIjSourceFolder.java

```diff
@@ -32,7 +32,12 @@
 
   public abstract boolean getIsTestSource();
 
-  public abstract boolean getIsAndroidResources();
+  public abstract boolean getIsResourceFolder();
+
+  public abstract ResourceFolderType getResourceFolderType();
```

Comment:
The fields are used by the template (ij-module.st); since IjSourceFolder is generated, I need the getter here to create the field.